src/influxdb/point_value.cr

```diff
@@ -51,6 +51,20 @@ module InfluxDB
     end
 
     private def escape_value(value, quote_escape)
```

Comment:
I'd rename that `format_value` or something, makes more sense since adding an `i` for integers is not really "escaping"